src/endpoints/PricingTables/types.ts

```diff
@@ -0,0 +1,63 @@
+export interface PricingTable {
+    options: Option[];
+}
+
+export enum BillingCycle {
+    MONTH = 'month',
+    YEAR = 'year',
+}
+
+export enum Currency {
+    EUR = 'eur',
+    USD = 'usd',
+}
+
+export enum OptionId {
+    STARTER = 'starter',
+    CORE = 'core',
+    PREMIUM = 'premium',
+}
+
+export enum LimitDisplayName {
+    STORIES = 'Stories',
+    USERS = 'Users',
+    SITES = 'Sites',
+    CONTACTS = 'Contacts',
+    EMAIL_SENDS = 'Email sends',
+    CUSTOM_SENDER_ADDRESSES = 'Custom sender addresses',
+}
+
+export interface Option {
+    id: `${OptionId}`;
+    display_name: string | null;
+    description: string | null;
+    overview_features_list: PlanFeatureRef[];
+    limits: Limit[];
+    prices: Price[];
+    add_ons: AddOn[];
+}
+
+export interface PlanFeatureRef {
+    name: string;
+    details?: string;
+}
+
+export interface Limit {
+    id: string;
+    display_name: `${LimitDisplayName}`;
+    value: null | number;
+    per: null | string;
+}
+
+export interface Price {
+    billing_cycle: `${BillingCycle}`;
+    currency: `${Currency}`;
+    unit_amount: number;
+    unit: string;
```

Comment:
Unit can be `null`, in case of agency plans for example.

Reply:
Maybe I brought confusion with `unit_amount` property? Do you think we should just call it `amount`?

Reply:
Yeap I believe `amount` makes more sense here.

Reply:
This is changed in the API.

Reply:
https://github.com/prezly/javascript-sdk/pull/215/commits/dfd2002b381e9b832b6077669f586ea6a27ea78b